.github/workflows/trigger-e2e-tests.yml

```
@@ -152,10 +175,63 @@ jobs:
+            ./runner.sh --run e2e --env ${{ matrix.env }} --elk ${{ matrix.version }} --ror-es ${{ env.ROR_ES_VERSION }} --ror-kbn ${{ env.ROR_KBN_VERSION }} --mode ${{ env.MODE }}
+        env:
+          ROR_ACTIVATION_KEY: ${{ secrets.ROR_KBN_LICENSE }}
+
+      - name: Stop Docker memory monitor
+        if: always()
+        uses: ./.github/docker-memory-monitor
+        with:
+          action: stop
+      - name: S3 Upload Videos & show logs
+        if: failure()
+        uses: ./.github/upload-videos
```

actionlint (1.7.12)

[error] missing input "access_key_id" which is required by action "Upload Videos" defined at "./.github/upload-videos". all required inputs are "access_key_id", "secret_access_key"

(action)